base/libgit2/types.jl

```diff
@@ -705,13 +705,19 @@ function getobjecttype(obj_type::Cint)
     end
 end
 
+import Base.securezero!
+
 "Credentials that support only `user` and `password` parameters"
 type UserPasswordCredentials <: AbstractCredentials
     user::AbstractString
```

Comment:
It strikes me that these fields should probably be `String`, but that's a question for another PR.